README.md

```diff
@@ -44,7 +44,7 @@ Then:
 
 ```bash
 > pip install --no-binary :all: root_numpy 
-> pip uninstall hawc_hal -y ; pip install git+https://github.com/threeml/hawc_hal.git
+> pip uninstall hawc_hal -y ; pip install git+https://github.com/torresramiro350/hawc_hal.git
```

Comment:
Could you change this back to installing from the official repo, please?

Reply:
Done.